### ISSUE TITLE
class heritage MessageTray.NotificationBanner is not an object or null

### ISSUE THREAD

**Issue by philipraets** [GSConnect/gnome-shell-extension-gsconnect#1762]
### Describe the bug

GSConnect fails to start on GNOME46, in the logs there is the following error:

```
Extension gsconnect@andyholmes.github.io: TypeError: class heritage MessageTray.NotificationBanner is not an object or null
@file:///home/%USER%/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/shell/notification.js:47:4
```

### Steps to reproduce

Install latest version from Github
Restart gnome-shell

### GSConnect version

Latest from github

### Installed from

GitHub releases (zip file)

### GNOME Shell version

46.0

### Linux distribution/release

opensuse Tumbleweed

**Comment by mendres82** [comment]
I encountered the same bug yesterday, after installing Gnome 46 on openSUSE Tumbleweed.

Fortunately, the issue was pretty easy to fix. I just had to edit _./shell/notification.js_, add a line

`import * as Calendar from 'resource:///org/gnome/shell/ui/calendar.js';`

and change line 48 to

`}, class NotificationBanner extends Calendar.NotificationMessage {`.

So far, it's working fine.

**Comment by muhammad-ans** [comment]
This fix is causing issues  with notifications, No notification displays

**Comment by mendres82** [comment]
During my tests at some point I had a notification show up, but after some more testing today, I can replicate the behavior that no notifications are displayed when sending things back and forth between my devices.

Basically, I changed the above lines, because [this commit](https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/940b658071047fab8d1db18f301c59095b384f79) removed NotificationBanner from messageTray which was originally used by the extension. I'll probably have to dig deeper into the problem when I find some time.

**Comment by andyholmes** [comment]
Closed by #1767, along with #1764
